### DISCUSSION TITLE
Surround sound 5.1 / 7.1

### DISCUSSION THREAD

**Discussion by alexthedamager** [OvenMediaLabs/OvenMediaEngine#1267] [Help]
Is 5.1 audio supported in SRT or RTMP ingest and WebRTC playback with OME?

It looks like Dolby.io / Millicast supports ingest and playback of 5.1 audio: https://docs.dolby.io/streaming-apis/docs/broadcasting-surround-sound-with-obs

**Comment by getroot** [comment]
There have been discussions about this in the past.

https://github.com/AirenSoft/OvenMediaEngine/issues/199

It still appears to be experimental. Even Chrome doesn't officially advertise multiopus, and other browsers don't support it. If there has been a recent change in browsers and standardization, please let me know.
